README.md

```diff
@@ -185,7 +185,9 @@ Want to see a more advanced example? [Checkout our test micro kernel](Tests/Fixt
 dunglas_action:
     autodiscover:         # Autodiscover action classes stored in the configured directory of bundles and register them as service.
         enabled:   true
-        directory: Action # The directory name to autodiscover in bundles.
+        directories: # The directories name to autodiscover in bundles.
+            action: [ Action ]
+            command: [ Command ]
```

Comment:
It would be easier to fix in the core as the issue is caused by the framework bundle. 
BTW I opened a PR which should fix this issue for this party bundle https://github.com/symfony/symfony/pull/18022